projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -553,26 +553,43 @@ public function get_jwt() {
 		// Only encrypt the attributes field as it contains sensitive information
 		// Content, hash, and source are not sensitive and can remain unencrypted
 
-		// Check cipher availability with fallback support. We store and compare
-		// cipher names in lowercase so that we behave consistently across
-		// environments where OpenSSL reports methods in uppercase.
-		$cipher                   = 'aes-256-gcm';
-		$available_cipher_methods = array_map( 'strtolower', openssl_get_cipher_methods() );
+		// Check cipher availability with fallback support
+		$available_cipher_methods = openssl_get_cipher_methods();
+		$cipher                   = null;
 		$use_encryption           = false;
 		$iv_length                = 12; // Default for GCM
 
```

Comment:
Do we need to continue to use `$use_encryption`? Couldn't we rely on checking if `$cipher` is null instead?